### ISSUE TITLE
Error registering workflow run undefined/api/workflow-runs

### ISSUE THREAD

**Issue by gloelmat** [garden-io/garden#2226]
### Current Behavior

We are using a garden workflow (that publishes Docker images to AWS ECR) as part of our cicd on github actions. Since yesterday afteroon we are getting the following error message (see screenshot below).
We have not changed anything about the workflow setup. However, we noticed that the garden version was updated to 0.12.13 yesterday. 

An error occurred while registering workflow run: Invalid URL: undefined/api/workflow-runs
Invalid URL: undefined/api/workflow-runs

![Bildschirmfoto 2021-01-21 um 14 21 00](https://user-images.githubusercontent.com/63103031/105357313-ee1da480-5bf4-11eb-925a-aca742648331.png)

### Expected behavior
This is how a normal run looks like:

![Bildschirmfoto 2021-01-21 um 14 32 54](https://user-images.githubusercontent.com/63103031/105357766-93d11380-5bf5-11eb-945f-c97c238ebaa4.png)
### Workaround

We reverted to version 0.12.12 and it works (!)  but we would like to stick to the latest version.
### Your environment

* **OS:** 
 Ubuntu-20.04 (github actions)

* **How I'm running Kubernetes:**  

latest minikube v1.16.0
latest garden 0.12.13 and later
```
jobs:
  build:
    runs-on: Ubuntu-20.04
    steps:
      - uses: actions/checkout@v2

      - name: Install Minikube
        run: |
          bash install/minikube.sh

      - name: Install Garden
        run: |
          bash install/garden.sh

      - name: Set Timestamp as Environment Variable
        run: |
          env_var=$(date "+%Y%m%d-%H%M%S")
          echo "BUILD_TIMESTAMP=$env_var" >> $GITHUB_ENV


      - name: "Garden Run: Deploy and Test to our Dev env"
        run: |
          export PATH=$PATH:$HOME/.garden/bin
          garden run workflow publish-workflow --env=ci-cd-deploy-our-dev --logger-type=basic

```

`garden version`

**Comment by 10ko** [comment]
Hi @gloelmat, thanks for the detailed report.
We are currently investigating the issue and we'll keep you posted.

**Comment by 10ko** [comment]
@gloelmat this is now fixed with [`0.12.15`](https://github.com/garden-io/garden/releases/tag/0.12.15).
Thanks again for reporting.
